Docs 719: combined TPC updates (#1035)
hazelcast/hz-docs: docs/modules/cluster-performance/pages/performance-tuning.adoc

= Performance Tuning

> To achieve good performance in your Hazelcast deployment, it is crucial to tune your

Reviewer: ```suggestion
You must tune your production environment to achieve the best performance in your Hazelcast deployment.
```

Author: Had a minor fix to this in main already so resolving...

> production environment. This section provides guidelines for tuning the performance though we also 

Reviewer: ```suggestion
This section provides guidelines for tuning the performance. In combination with these guidelines, we 
```

> recommend to run performance and stress tests to evaluate the application performance.

Reviewer: ```suggestion
recommend that you run performance and stress tests to evaluate the application performance.
```
